src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,108 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+
+    @BeforeEach
+    public void setUp() {
+        user = new User();
+        user.setAge(18);
+        user.setLogin("123456");
+        user.setPassword("123456");
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_ok() {
+        registrationService.register(user);
+        User stored = Storage.people.get(0);
+        assertNotNull(stored, "User should be stored in Storage");
+    }
+
+    @Test
+    public void register_someUsers_ok() {
+        User user2 = new User();
+        user2.setLogin("secondLogin");
+        user2.setPassword("secondPassword");
+        user2.setAge(31);
+        User user3 = new User();
+        user3.setLogin("thirdLogin");
+        user3.setPassword("thirdPassword");
+        user3.setAge(21);
+        registrationService.register(user);
+        registrationService.register(user2);
+        registrationService.register(user3);
+        boolean expected1 = Storage.people.get(0).equals(user);
+        boolean expected2 = Storage.people.get(1).equals(user2);
+        boolean expected3 = Storage.people.get(2).equals(user3);
+        assertTrue(expected1, "First user wasn't added");
+        assertTrue(expected2, "Second user wasn't added");
+        assertTrue(expected3, "Third user wasn't added");
```

Comment:
The method name 'register_someUsers_ok' does not follow the naming convention from the checklist. It should be something like 'register_multipleValidUsers_ok'. Also, consider using `storageDao.add(user)` directly instead of `registrationService.register(user)` to add users for testing.